### ISSUE TITLE
Error in H5F

### ISSUE THREAD

**Issue by AviMaayan** [MaayanLab/archs4dx#28]
# Retrieve information from compressed data
> samples = h5read(destination_file, "meta/samples/geo_accession")
Error in H5Fopen(file, flags = flags, fapl = fapl, native = native) : 
  HDF5. File accessibility. Unable to open file.

**Comment by lachmann12** [comment]
Working on R package should help with R user problems
